the-events-calendar.php

```diff
@@ -29,7 +29,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-define( 'TRIBE_EVENTS_FILE', __FILE__ );
+define('TRIBE_EVENTS_FILE' , __FILE__ );
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Space found before comma in function call

<sub>hash: 1193e102a9768ca9e0cf8f94dc4047ac</sub>